src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -710,6 +715,13 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
     noshort = true,
   )
 
+  val disableCheckTerminationPureFns: ScallopOption[Boolean] = opt[Boolean](
+    name = "disableTermCheckPureFns",
```

Comment:
```suggestion
    name = "disablePureFunctsTerminationRequirement",
```

Reply:
I'm definitely open to other suggestions but that's one I'd consider better than the current one.
Some of my arguments:
- `Term` is imho too short as one could confuse it with, e.g., a term algebra we use for our work on security protocols
- `Check` is imho confusing as enabling this option will not reduce proof obligations since pure functions with termination measures will still be checked for termination